app/src/components/wallet/PageTransactions.vue

```diff
@@ -1,17 +1,13 @@
 <template>
   <TmPage
-    :managed="true"
-    :loading="$apollo.queries.transactions.loading && dataEmpty"
-    :loaded="!$apollo.queries.transactions.loading && !dataEmpty"
-    :error="$apollo.queries.transactions.error"
-    :data-empty="transactions.length === 0"
     data-title="Transactions"
+    :loading="$apollo.queries.transactions.loading"
+    :error="$apollo.queries.transactions.error"
     :sign-in-required="true"
-    :hide-header="true"
   >
-    <DataEmptyTx slot="no-data" />
-    <template slot="managed-body">
-      <div>
+    <template slot="signInRequired">
+      <DataEmptyTx v-if="!$apollo.queries.transactions.loading && dataEmpty" />
```

Comment:
add data empty to TmPag?

Reply:
`DataEmptyTx` is just for transactions. i guess we should put a `DataEmpty` into `TmPage` and use props for the specific wording / language?